conductor/submitter_maya.py

```diff
@@ -24,6 +24,7 @@
 except ImportError, e:
     sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
+reload(submitter)
```

Comment:
Would be great to have a comment to say why this is needed.

Reply:
@hoolymama Oops.. I had committed the wrong file. I pushed the correct changes.